src/rapi/rc_api_common.c

```diff
@@ -388,7 +397,7 @@ static unsigned rc_json_decode_hex4(const char* input) {
   memcpy(hex, input, 4);
   hex[4] = '\0';
 
-  return strtol(hex, NULL, 16);
+  return (unsigned)strtol(hex, NULL, 16);
```

Comment:
Any reason not to use `strtoul`?

Reply:
Nope. Hex numbers (base 16) can't be signed. And this function is specifically for decoding 16-bit hex numbers. The downcast to `unsigned [int]` is still required as `strtoul` returns `unsigned long`.